Log Viewing/ARKLogTableViewController.m

```diff
@@ -481,10 +481,6 @@ - (void)_scrollTableViewToBottomAnimated:(BOOL)animated;
 
 @implementation ARKTimestampLogMessage
 
-@synthesize text = _text;
```

Comment:
these are needed because `ARKTimestampLogMessage` is declared in `ARKLogTableViewController_Testing.h`, and not in `ARKLogTableViewController`'s .m or .h.